crmsh/ui_node.py

```diff
@@ -9,9 +9,99 @@
 from . import utils
 from . import xmlutil
 from .msg import common_err, syntax_err, no_prog_err, common_info, common_warn
```

Comment:
should be `common_error` here, or have errors like `NameError: name 'common_error' is not defined`